hrjobcontract/CRM/Hrjobcontract/Page/PayScale.php

```diff
@@ -120,6 +120,7 @@ function browse() {
       //Not Applicable pay grade should not be deleted
       if ($dao->pay_scale === 'Not Applicable') {
         $action -= CRM_Core_Action::DELETE;
+        $action -= CRM_Core_Action::UPDATE;
```

Comment:
I think you need to add "and not edited" to the inline comment above to reflect this case .. also note my previous comment about refactoring generating links into separate method